lib/rules/no-forbidden-elements.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const Rule = require('./base');
+const createErrorMessage = require('../helpers/create-error-message');
+
+function ERROR_MESSAGE_FORBIDDEN_ELEMENTS(element) {
+  return `Use of <${element}> detected. Do not use forbidden elements.`;
+}
+
+const FORBIDDEN_ELEMENTS = ['meta', 'style', 'html', 'script'];
+
+const DEFAULT_CONFIG = {
+  forbidden: [],
+};
```

Comment:
The `DEFAULT_CONFIG` should be where the list of default forbidden elements live:

```suggestion
const DEFAULT_CONFIG = {
  forbidden: ['meta', 'style', 'html', 'script'],
};
```

Reply:
Done